packages/datetime/package.json

```diff
@@ -13,9 +13,10 @@
   },
   "devDependencies": {
     "bourbon": "4.2.6",
-    "react": "^15.0.1 || ^0.14",
```

Comment:
this was the source of the build failures -- React 15.4 was being used with react-addons-test-utils 15.1. These versions are incompatible with each other. Moral of the story: _don't declare floating version ranges in `devDependencies`.